### DISCUSSION TITLE
Make "mute" option.

### DISCUSSION THREAD

**Discussion by zhvial** [micbay/RaceTimerAndController#5] [Ideas]
It is not possible to turn off sound. The sound from buzzer sometimes could be too loud. 
For now I have made "mechanical" switch to turn of buzzer sound. It would be good to have an option to mute music effects from menu.  May be the siplest solution would be to use # key . Press once - mute music, but leave lap passing sound. Press twice - mute all.

**Comment by micbay** [comment]
It would be a nice option to be able to set a sound mode like of 'all', 'no music', and 'mute', and adjust it from the UI. As with any feature requiring new code and variables, limited remaining memory must be considered vs the value added.
However, I don't think this would add that much and is worth taking a look at in a test branch.

**Comment by micbay** [comment]
As of the current master commit (Jan 5, 2023 5pm EST USA), audio modes have been implemented. The current active audio can be seen in the 'Change Settings' sub-menu and toggled through various states from that screen.
Possible audio modes are
- AllOn - Game/UI beeps and boops are on, and game music is on
- GameOnly - Game/UI beeps and boops are on, but the racer songs will not play
- Mute - no audio will play

Game Audio active is identified by a speaker icon
Music Audio active is identified by a music note icon
In the Mute Audio Mode, the audio setting is declared '-OFF-'

**Comment by zhvial** [comment]
Great! 
Thanks for fast implementation. 

By the way, I have added beeps to coundown timer before race starts.
I've added beep() to the count down cycle, where led screen counts 11111111, 2222222, 3333333. And boop(), when "start" is shown. So now you can also hear beep beep beep --- boop.

**Comment by micbay** [comment]
Another good idea. I've added this to the main code base, with a new 'LongBoop()' tone for the 'Start' trigger.

The latest code base now has this feature added, as well as a new `defaultSettings.h` and `example.localSettings.h` file that can be copied and turned into a `localSettings.h` file that can be used to override the `defaultSettings` with any UI text translation and controller setup settings users want to customize. This puts all the main customizable settings in one location, and provides a place to store them that won't get overwritten with future updates to the root `RaceTimeAndController.ino` file.
